### PR TITLE
Use order list in section 2.1

### PR DESCRIPTION
**Pull request by OR13**
This PR attempts to make the examples more readable.

I was not able to lint or validate these changes locally.

### CODE REVIEW

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, merging. Thank you!

File: index.xml [hunk 1]
```diff
@@ -136,17 +136,23 @@ as the modified media type further down the pipeline.
...
-(as specified in the  <eref target="https://www.iana.org/assignments/media-types/media-types.xhtml">Media Type Registry</eref>),
-2) +ld+json (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>),
-or 3) +json (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>).
+following processing models:
+      </t>
+      <ol>
+        <li> application/did+ld+json (as specified in the  <eref target="https://www.iana.org/assignments/media-types/media-types.xhtml">Media Type Registry</eref>),</li>
+        <li> +ld+json (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>),</li>
+        <li> +json (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>).</li>
```
Comment (TallTed) [inline index.xml:144]: Merged too quickly!
```suggestion
        <li> <code>application/did+ld+json</code> (as specified in the  <eref target="https://www.iana.org/assignments/media-types/media-types.xhtml">Media Type Registry</eref>)</li>
        <li> <code>application/ld+json</code> (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>)</li>
        <li> <code>application/json</code> (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>)</li>
```

File: index.xml [hunk 2]
```diff
@@ -136,17 +136,23 @@ as the modified media type further down the pipeline.
...
-following processing models: 1) image/svg+xml+gzip
-(as specified in the  <eref target="https://www.iana.org/assignments/media-types/media-types.xhtml">Media Type Registry</eref>),
-2) +gzip (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>),
-and then +xml (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>).
+following processing models:
       </t>
+      <ol>
+        <li> image/svg+xml+gzip (as specified in the  <eref target="https://www.iana.org/assignments/media-types/media-types.xhtml">Media Type Registry</eref>),</li>
+        <li> +gzip (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>),</li>
+        <li> +xml (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>).</li>
```
Comment (TallTed) [inline index.xml:154]: Merged too quickly!
```suggestion
        <li> <code>image/svg+xml+gzip</code> (as specified in the  <eref target="https://www.iana.org/assignments/media-types/media-types.xhtml">Media Type Registry</eref>),</li>
        <li> <code>application/gzip</code> (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>),</li>
        <li> <code>application/xml</code> _or is that <code>text/xml</code>?_ (as specified in the <eref target="https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml">Structured Syntax Suffixes Registry</eref>).</li>
```

Comment (OR13) [reply index.xml:154]: this is in xml, not sure if the code block is supported.

**Comment by TallTed** [comment]
This PR takes this RFC further into disagreement with existing RFC and related rules. The second ordered list is particularly problematic. It should not have been merged.

**Pull request merged**